tests/plotting/test_plotting.py

```diff
@@ -244,6 +245,19 @@ def test_pbr(sphere):
     pl.show(before_close_callback=verify_cache_image)
 
 
+@kip_not_vtk9
```

Comment:
Time for bed, sorry.
```suggestion
@skip_not_vtk9
```